test.mocha.js

```diff
@@ -31,3 +31,10 @@ sd.AP = {
 }
 sd.APP_URL = "http://artsy.net"
 sd.RECAPTCHA_KEY = "RECAPTCHA_KEY"
+
+global.stitch = {
+  components: {
+    NavBar: x => x,
+    Footer: x => x,
+  },
+}
```

Comment:
@damassi Curious for your thoughts on adding this to our global Mocha setup? If someone needs to test these individual components they can always overwrite it in the actual test.

Reply:
All for it 👍 

When someone _wants_ to test these things, then we can have the discussion.